static/styles/right_sidebar.css

```diff
@@ -98,13 +100,31 @@
     overflow: hidden;
 
     .user-name {
+        max-width: 130px;
```

Comment:
Why this value?

It would be helpful if you're adding a bunch of numbers like this to define variables at the top of the file, like we do for `left-sidebar.css`, and do math with them, to make it easier to understand.